packages/react-native-reanimated/src/ReanimatedModule/js-reanimated/index.ts

```diff
@@ -1,38 +1,32 @@
 'use strict';
-import JSReanimated from './JSReanimated';
-import type { StyleProps, AnimatedStyle } from '../commonTypes';
+import type { StyleProps, AnimatedStyle } from '../../commonTypes';
 import {
   createReactDOMStyle,
   createTransformValue,
   createTextShadowValue,
 } from './webUtils';
-import { PropsAllowlists } from '../propsAllowlists';
-import { logger } from '../logger';
-import { ReanimatedError } from '../errors';
+import { PropsAllowlists } from '../../propsAllowlists';
+import { logger } from '../../logger';
+import { ReanimatedError } from '../../errors';
 
-const reanimatedJS = new JSReanimated();
+export { createJSReanimatedModule } from './JSReanimated';
 
+// TODO: Install these global functions in a more suitable location.
```

Comment:
We'll need to move it to a separate file and also mark as `sideEffects` in package.json.

Reply:
I'll make a note for myself to add some ESLint plugin for sideEffect detection in the future.